src/lib/dom.generated.d.ts

```diff
@@ -12896,28 +12432,6 @@ declare var SVGElement: {
     new(): SVGElement;
 };
 
-interface SVGElementInstance extends EventTarget {
-    readonly correspondingElement: SVGElement;
-    readonly correspondingUseElement: SVGUseElement;
-}
```

Comment:
These properties used to be available on `SVGElement`, which used to extend `SVGElementInstance` in [LOC 12883](https://github.com/microsoft/TypeScript/pull/44684/files#diff-dc0eab937d15e62545da3ed7b4f40ad6b24f15dd88fbc6ceda2bfb4ed8356eb0L12883). They disappeared during the upgrade from [`v4.3.5` to `v4.4.2`](https://github.com/microsoft/TypeScript/compare/v4.3.5...v4.4.2).

Reply:
Removal was intentional, as per https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1023#issuecomment-908524091:

> Those are not available on any modern browser nor MDN, and thus are probably IE-specific properties. The removal is intended since the types library intends to only include things that exist in modern browsers.